src/Integration.Vsix.UnitTests/Commands/PackageCommandManagerTests.cs

```diff
@@ -38,32 +36,6 @@ namespace SonarLint.VisualStudio.Integration.UnitTests
     [TestClass]
     public class PackageCommandManagerTests
     {
-        private ConfigurableServiceProvider serviceProvider;
```

Comment:
Most of this rubbish wasn't required so I removed it.
I removed the `[TestInitialize]` here because it was simple to do so.